client/src/app/models/site.coffee

```diff
@@ -0,0 +1,10 @@
+Backbone = require('backbone')
+
+module.exports = Backbone.Model.extend(
+  urlRoot: 'http://localhost:8989/sites'
```

Comment:
You can set up a dev server proxy so server requests can be made with relative urls. Are you using the webpack dev server or some gulp server task? Should be possible with either.